src/explore-education-statistics-admin/src/components/WysiwygEditor.tsx

```diff
@@ -128,22 +130,25 @@ const WysiwygEditor = ({
           <CKEditor
             editor={ClassicEditor}
             config={{
-              toolbar: [
-                'heading',
-                '|',
-                'bold',
-                'italic',
-                'link',
-                '|',
-                'bulletedList',
-                'numberedList',
-                '|',
-                'blockQuote',
-                'insertTable',
-                '|',
-                'redo',
-                'undo',
-              ],
+              toolbar:
+                toolbarStyle === 'full' || toolbarStyle === undefined
+                  ? [
+                      'heading',
+                      '|',
+                      'bold',
+                      'italic',
+                      'link',
+                      '|',
+                      'bulletedList',
+                      'numberedList',
+                      '|',
+                      'blockQuote',
+                      'insertTable',
+                      '|',
+                      'redo',
+                      'undo',
+                    ]
+                  : ['bold', 'link', '|', 'bulletedList'],
```

Comment:
Let's allow the toolbar config array to be passed as a prop, that way we can define exactly which controls we want.
By default it can fallback to the 'full' config outlined here, but then we can pass `toolbar={['heading', '|', 'link']}` or any other controls we want..

Reply:
I agree with the other above. A good addition would be to also export a bunch of prebaked defaults that we can use in other components. We want to have flexibility whilst avoiding too many inconsistencies in the UIs.